src/main/java/com/rultor/agents/daemons/EndsDaemon.java

```diff
@@ -155,6 +168,30 @@ public String apply(final String str) {
                     )
                 )
             );
+        EndsDaemon.deprecate(directives, deprecate);
+        return directives;
+    }
+
+    /**
+     * Adds the deprecated message to the directive.
+     * @param directives the directives which being deprecated
+     * @param deprecate true if it is deprecated.
+     */
+    private static void deprecate(final Directives directives,
+        final boolean deprecate) {
+        if (deprecate) {
```

Comment:
@mkordas @sebing no sorry, I mean you must agree that this would be as wrong as it gets right? :/
Merging this would simply mean that we merge something completely untested and try it out in production, can't have that on my conscience.
http://www.yegor256.com/2009/03/04/pdd.html#best-practices explains the way this needs to go pretty well imo, make the test and add the puzzle for the production code. The other way around is not ok.